tests/checkBoxes.spec.ts

```diff
@@ -36,10 +38,12 @@ test("Validate selected specialties", async ({ page }) => {
 });
 
 test("Select all specialties", async ({ page }) => {
-  const veterinarianRafael = page.locator("tr", {
-    has: page.getByText(" Rafael Ortega "),
-  });
-  await veterinarianRafael.getByRole("button", { name: "Edit Vet" }).click();
+  await page
+    .locator("tr", {
+      has: page.getByText(" Rafael Ortega "),
+    })
+    .getByRole("button", { name: "Edit Vet" })
+    .click();
```

Comment:
Why you don't make it as a single line and stretch it to 6 lines of code? It looks a little bit weird.

Reply:
I'm using `Format Document` command. After that got this result.
Maybe you can recommend another specific prettier config?

Reply:
https://github.com/AlexxDk/playwright-homework/assets/38554790/c64ac87e-fd28-4dee-bb11-bd2be5d3154b

actually I do the same ^

Reply:
Right, but you are using Prettier but I use a default formatter.

Reply:
ok, then I will with to this one. Seems it is default from VS code
<img width="622" alt="Знімок екрана 2024-06-10 о 20 26 33" src="https://github.com/AlexxDk/playwright-homework/assets/38554790/71836384-0b65-4bde-8643-23faf43252af">

Reply:
Yeah, but it will not format commands into a single line. All it does is correct spacings and tabs. Also, it formats JSON files correctly. So If you want to change the commands to single-line, you ll need to make it manually.